lightning.services.yml

```diff
@@ -44,3 +44,10 @@ services:
     arguments: []
     tags:
       - { name: drupal.generator }
+
+  lightning.version_command:
+    class: 'Drupal\lightning\Command\VersionCommand'
+    arguments:
+            - '@app.root'
```

Comment:
Nit: Whitespace counts in YAML, so we need to outdent the argument a few times :)

Reply:
darn good eye, sir.